consensus/miner_signalled_consensus.md

```diff
@@ -0,0 +1,200 @@
+# Miner Signalled Consensus Upgrade
+
+## Simple Summary
+
+This document describes the proposal of how nodes in the peer-to-peer network
```

Comment:
--> This document describes a protocol to signal the readiness for, and activation of, a new hard-fork at a certain height.